app/src/main/java/com/example/android/picover/presentation/navigation/PicoverNavHost.kt

```diff
@@ -24,10 +25,11 @@ fun PicoverNavHost(
 			HomeScreen()
 		}
 		composable("camera") {
-			CameraScreen()
+			CameraScreen(hiltViewModel())
```

Comment:
Minor: You could pass `hiltViewModel()` as the default argument in `CameraScreen`.